File: Number-of-Laser-Beams-in-a-Bank/Number-of-Laser-Beams-in-a-Bank.py
bank = ["011001","000000","010100","001000"]

def numberOfBeams(bank):
    l = 0
    sums = 0

    for row in bank:
        if "1" in row:
            r = row.count("1")
            sums += l * r
            l = r

    return sums

print(numberOfBeams(bank))

File: Number-of-Laser-Beams-in-a-Bank/Number-of-Laser-Beams-in-a-Bank.js
let bank = ["011001","000000","010100","001000"]

var numberOfBeams = function(bank) {
    let l = 0
    let sums = 0

    for(let i = 0 ; i<bank.length;i++){
        if(bank[i].includes("1")){
            let r = bank[i].split("1").length - 1
            sums = sums + l*r
            l=r
        }
    }

    return sums
};

console.log(numberOfBeams(bank))